### ISSUE TITLE
Wikipedia link points to English instead of app language

### ISSUE THREAD

**Issue by Theklan** [inaturalist/SeekReactNative#258]
When we see try to read more information about a species we have found a short text from Wikipedia is shown (if the name of the species match the one at Wikipedia). But the link to read more goes to English Wikipedia instead of the app language Wikipedia. This is problematic for children, as most of them don't speak English.

It should point to the correct Wikipedia version.

**Comment by kueda** [comment]
FWIW, a trivial solution that would cover most situations would just be to replace the `en.` part of the URL with the current language part of the locale, but this is really a problem we should solve in the web app since it affects all parts of the platform where we link to Wikipedia. Wikidata maintains a mapping between iNat taxon identifiers and Wikipedia names in all languages so this is definitely possible, but will take some finagling. Discussion of relevant functionality is at https://forum.inaturalist.org/t/use-wikidata-to-link-to-appropriate-wikipedia-articles-in-all-languages/5538/20

**Comment by Theklan** [comment]
Thanks @kueda. The solution you are proposing is the easiest one, and can be done without changin all the system using Wikidata (AFAIK, the best solution).

**Comment by albullington** [comment]
I took a quick pass at updating the `en.` part of the URL in Seek, and one thing I noticed is that a lot of the species I tested were returned from the iNat API with the English common name in the URL (for example, `http://en.wikipedia.org/wiki/Snowy_egret` or `http://en.wikipedia.org/wiki/Mule_deer`), which makes this a little less trivial.

We could try ignoring the iNat wiki URL altogether when the app is in a different language, and instead we could create a new Wikipedia URL within Seek using the locale and the scientific name (i.e. `https://es.wikipedia.org/wiki/Odocoileus_hemionus`), though I imagine that will be a lot less reliable than using the Wikidata mappings.
